sources/packages/web/src/views/aest/StudentApplicationView.vue

```diff
@@ -209,13 +260,78 @@ export default defineComponent({
       document.getElementById(component.id)?.classList.add(cssClass);
     }
 
+    const assessApplicationChangeRequest = async (
+      applicationChangeRequestStatus: ApplicationEditStatus,
```

Comment:
Please type it with only relevant values. 
![image](https://github.com/user-attachments/assets/20fd84e3-d6d8-4231-ae72-5608e6dbb0f1)